src/Test/SparseVector/SparceVector.cs

```diff
@@ -0,0 +1,97 @@
+namespace SparseVector;
+
+public class Vector
+{
+    private Dictionary<int, float> vector;
+
+    public Vector()
+    {
+        this.vector = new Dictionary<int, float>();
+    } 
```

Comment:
```suggestion
    private Dictionary<int, float> vector = new();
```
Инициализаторы лучше.